website/docs/r/route53_zone.html.markdown

```diff
@@ -40,17 +40,11 @@ resource "aws_route53_zone" "dev" {
 }
 
 resource "aws_route53_record" "dev-ns" {
-  zone_id = "${aws_route53_zone.main.zone_id}"
+  zone_id = aws_route53_zone.main.zone_id
   name    = "dev.example.com"
   type    = "NS"
   ttl     = "30"
-
-  records = [
-    "${aws_route53_zone.dev.name_servers.0}",
-    "${aws_route53_zone.dev.name_servers.1}",
-    "${aws_route53_zone.dev.name_servers.2}",
-    "${aws_route53_zone.dev.name_servers.3}",
-  ]
+  records = aws_route53_zone.dev.name_servers
```

Comment:
@bflad do you mind clarifying -- is the switch from declaring all the name servers to referencing the list in line with changes that you think should be made where available? If so, I'll try to utilize any shortcuts like this one in followup PRs to other resource examples.

Reply:
@DavidJFelix yes, its a good change and there will likely be other places where similar 0.12 optimizations can be done. Please do note though that we will be running an automated script as part of #8950 to update all the web page examples at once in the next week, so I would suggest waiting until that is ran and merged before submitting more 0.12 syntax updates. 👍

Reply:
Sounds good!